lib/single_sign_on.rb

```diff
@@ -1,8 +1,9 @@
 class SingleSignOn
   ACCESSORS = [:nonce, :name, :username, :email, :avatar_url, :avatar_force_update, :require_activation,
-               :about_me, :external_id, :return_sso_url, :admin, :moderator, :suppress_welcome_message]
+                :no_user_found_return_sso_url,:user_found, :about_me, :external_id, :return_sso_url,
```

Comment:
You have an additional blank here. The most important part, I suppose `return_sso_unlogged_in_url` or something else could be more clear and align with `return_sso_url`?